readme.md

```diff
@@ -287,6 +287,8 @@ If you're on Windows, do yourself a favor and use [`cmder`](http://cmder.net/) i
 
 ## Related
 
+- [chalkie](https://github.com/shawwn/chalkie) - Browser support for chalk
+- [ansi-html](https://github.com/Tjatse/ansi-html) - Convert chalked text to HTML
```

Comment:
I don't want to link to this one. It has lots of bugs and doesn't seem maintained.